packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md

```diff
@@ -113,13 +123,13 @@ The private key that you use to sign the JWT must have the corresponding public
     * Key size: 2048
     * Key use: signature
     * Algorithm: RSA256
-    * Key ID: This can be any random value.
+    * Key ID: This can be any random value
```

Comment:
for consistency with the previous bullets, I think you could just make this

any random value

(that way it's lowercase like "signature")...but up to you. I'm fine either way